facebook_page_scraper/element_finder.py

```diff
@@ -409,3 +472,40 @@ def __accept_cookies(driver):
         except Exception as ex:
             logger.exception("Error at accept_cookies: {}".format(ex))
             sys.exit(1)
+
+    @staticmethod
+    def __login(driver, username, password):
```

Comment:
Thanks for these improvements. About this, I think this might cause legal issues later on. I think we should avoid logged in scraping

Reply:
Maybe we can add warning that user do it at their own risk and they can get permanent ban